### PR TITLE
Abram

### PR DESCRIPTION
**Pull request by legoabram**

### CODE REVIEW

File: app/assets/javascripts/event-stage.js [hunk 1]
```diff
@@ -59,11 +96,42 @@
...
+                        methods: {
+                            isValid: function( participant ) {
+                                if ( !participant.first ) return false;
+                                return true;
+                            },
+                            blank: function() {
+                                return $.extend( {}, blanks.participant );
+                            }
+                        },
+                        data: function() { return { modalData: {}, filter: '' }; },
```
Comment (winterpk) [inline app/assets/javascripts/event-stage.js]: Un-necessary `;`

File: app/assets/javascripts/event-stage.js [hunk 2]
```diff
@@ -122,14 +198,11 @@
             onMounted: function() {
                 this._queue = [];
                 this._table = $( this.$el ).DataTable( {
-                    oLanguage: {
-                        'sSearch': 'Filter:&nbsp;'
-                    }
+                    dom:    "<'row'<'col-sm-12'tr>><'row'<'col-sm-5'i><'col-sm-7'p>>",
```
Comment (winterpk) [inline app/assets/javascripts/event-stage.js]: Woah nelly!

File: app/assets/javascripts/event-stage.js [hunk 3]
```diff
@@ -156,61 +229,165 @@
...
+                        var self = this;
+                        var reset = function() {
+                            // Locally clone existing object
+                            self.model = $.extend( {}, self.value );
+                        };
+                        $( this.$el ).on( 'show.bs.modal hidden.bs.modal', reset );
+                        this.$on( 'cancel', reset );
+                        this.$on( 'done', function() {
+                            // Copy properties to original object
+                            // for ( var prop in self.model ) {
```
Comment (winterpk) [inline app/assets/javascripts/event-stage.js:357]: Delete commented code if you can

File: app/assets/javascripts/utilities.js [hunk 4]
```diff
@@ -117,9 +117,13 @@
...
         effortsPopover.init();
         staticPopover.init();
         switchery.init();
-    });
+    };
+
+    $(document).ready( init );
+    $(document).bind( 'vue-ready', init );
+
 })(jQuery);
```
Comment (winterpk) [inline app/assets/javascripts/utilities.js]: Add a new line at the end of this file. Github complains if you don't

File: app/views/stage/index.html.erb [hunk 5]
```diff
@@ -80,56 +90,168 @@
...
+				<p class="pull-left">
+					<button type="button" class="btn btn-success" data-toggle="modal" data-target="#js-split-modal" v-on:click="modalData = blank()">Add New Split&nbsp;<i class="glyphicon glyphicon-plus"></i></button>
+					<button type="button" class="btn btn-default">Import GPX File</button>
+				</p>
+				<router-link to="/splits" class="btn btn-primary pull-right">Continue&nbsp;<i class="glyphicon glyphicon-arrow-right"></i></router-link>
+			</div>
 		</div>
 	</article>
-	<split-modal :isNew="modalNew" :split="modalSplit" v-on:new="eventData.splits.push( modalSplit )"></split-modal>
+	<edit-modal v-model="modalData" :validator="isValid" v-on:done="!( modalData._dtid ) && eventData.splits.push( modalData )" inline-template>
```
Comment (winterpk) [inline app/views/stage/index.html.erb:166]: Github is complaining about this && for some reason.  Probly thinks its HTML. C'mon Github...

File: app/assets/javascripts/event-stage.js (hunk 2 above)
Comment (legoabram) [reply app/assets/javascripts/event-stage.js]: Yeah... DataTables really doesn't like using standard conventions.

**Pull request merged**